examples/express-ts/package.json

```diff
@@ -14,24 +14,24 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.16.1",
-    "express": "^4.14.1",
-    "express-promise-router": "^1.1.1",
-    "knex": "^0.12.7",
+    "body-parser": "^1.17.2",
+    "express": "^4.15.3",
+    "express-promise-router": "^2.0.0",
+    "knex": "^0.13.0",
     "lodash": "^4.17.4",
-    "morgan": "^1.8.1",
-    "objection": "^0.7.0",
+    "morgan": "^1.8.2",
+    "objection": "../..",
```

Comment:
Actually I didn't notice that. I think that's not the best idea since the example may break if the master branch has breaking changes. I usually only update the examples with actual releases so that I don't have to constantly be changing them. Also if a breaking change is updated to the example project and then someone starts to use objection based on that example, then `npm install objection` will work different from the example.